app/models/sample.rb

```diff
@@ -34,10 +34,10 @@ class Sample < ApplicationRecord
   DEFAULT_MEMORY_IN_MB = 120_000 # sorry, hacky
   HOST_FILTERING_MEMORY_IN_MB = 240_000
 
-  DEFAULT_QUEUE = 'idseq'.freeze
+  DEFAULT_QUEUE = Rails.env == 'development' ? "idseq-staging-lomem" : "idseq-#{Rails.env}-lomem".freeze
```

Comment:
check with @jameshowardwang if there's not actually a development environment coming for AWS Batch?